include/ppx/grfx/grfx_enums.h

```diff
@@ -259,6 +259,22 @@ enum Filter
     FILTER_LINEAR  = 1,
 };
 
+enum FoveationMode
+{
+    FOVEATION_NONE = 0,
+    FOVEATION_FDM  = 1,
+    FOVEATION_VRS  = 2,
+};
+
+enum FoveationPatternType
```

Comment:
Also consider loading pattern from file, either consist of the foveation pattern data themselves or pixel per degree that we can interpolate&quantize to a pattern.

Reply:
SG. It's a suggestion and does not need to be in the PR.